api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -111,8 +151,6 @@ else if(person.getCauseOfDeath() == null && person.getCauseOfDeathNonCoded() ==
 				errors.rejectValue("causeOfDeath", "Person.dead.causeOfDeathAndCauseOfDeathNonCodedNull");
 			}
 		}
-		
```

Comment:
Did you intentionally remove this?

Reply:
I did not remove line 114 and 115. I put it down where it was before I changed the code according to the previous pull request. Git Change Log does not recognize it this way though:
https://github.com/openmrs/openmrs-core/pull/2953/commits/4c40dade91b459e578231a2e593828832a5b51d9#diff-7014d6c9eb6084f32ffc5c659e8ee524R87